docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -8,9 +8,10 @@ Spring Framework on Google Cloud has dependency and transitive dependencies on S
 |===
 | Spring Framework on Google Cloud | Spring Cloud | Spring Boot | Spring Framework
 
-|2.x | https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2020.0-Release-Notes[2020.0.x] (3.0/Illford) |2.4.x, 2.5.x | 5.3.x
-|3.x | https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2021.0-Release-Notes[2021.0.x] (3.1/Jubilee) |2.6.x, 2.7.x | 5.3.x
-|4.x | https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2022.0-Release-Notes[2022.0.x] (4.0/Kilburn) |3.0.x,3.1.x  | 6.0.x
+|5.x | https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2023.0-Release-Notes[2023.0.x] (Leyton) |https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.2-Release-Notes[3.2.x]| https://github.com/spring-projects/spring-framework/wiki/What%27s-New-in-Spring-Framework-6.x#whats-new-in-version-61[6.1.x]
+|4.x | https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2022.0-Release-Notes[2022.0.x] (Kilburn) |https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Release-Notes[3.0.x], https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.1-Release-Notes[3.1.x]| https://github.com/spring-projects/spring-framework/wiki/What%27s-New-in-Spring-Framework-6.x#whats-new-in-version-60[6.x]
+|3.x | https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2021.0-Release-Notes[2021.0.x] (Jubilee) |2.6.x, 2.7.x | 5.3.x
+|2.x | https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2020.0-Release-Notes[2020.0.x] (Ilford) |2.4.x, 2.5.x | 5.3.x
```

Comment:
Is `2.x` still supported?

Reply:
If it's not supported anymore, why don't we remove it?

Reply:
I've added a 'Supported' column to this table (like the main readme has) to make it clear that 2.x is no longer supported. I'd rather not go further in this PR towards making a decision on removing 2.x references.